web/src/elements/Base.ts

```diff
@@ -68,7 +68,8 @@ export class AKElement extends LitElement {
         if ("ShadyDOM" in window) {
             styleRoot = document;
         }
-        styleRoot.adoptedStyleSheets = [...styleRoot.adoptedStyleSheets, AKGlobal];
+        const globalStyleSheet = AKGlobal instanceof CSSResult ? AKGlobal.styleSheet : AKGlobal;
```

Comment:
Depending on this page is loaded, this check is needed.  Storybook isn't so tolerant of poorly specified APIs as some other features might be.